src/rgw/rgw_admin.cc

```diff
@@ -2184,7 +2184,11 @@ class BucketReshardManager {
   }
 };
 
-int main(int argc, char **argv)
+#ifdef BUILDING_FOR_EMBEDDED
+extern "C" int cephd_rgw_admin(int argc, const char **argv)
```

Comment:
so this is basically just a pass through for the main entry point of the rgw-admin tool?